appservice/src/intTest/groovy/nu/yona/server/SystemAdminMessagesTest.groovy

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server
+
+import groovy.json.*
+import nu.yona.server.test.AdminService
+
+class SystemAdminMessagesTest extends AbstractAppServiceIntegrationTest
+{
+	def AdminService adminService = new AdminService()
```

Comment:
I propose to move this up to ``AbstractAppServiceIntegrationTest``, just in case other tests also want to use this service.

And annotate it with ``@Shared``.